src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage {
+    public static final Map<String, Integer> fruitStorage = new HashMap<>();
```

Comment:
The `fruitStorage` map is public and mutable, which can lead to unintended modifications from outside the class. Consider making the map private and providing controlled access through methods to ensure encapsulation and data integrity.